context/app/static/js/components/workspaces/ConfirmDeleteWorkspacesDialog/ConfirmDeleteWorkspacesDialog.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Divider,
+  IconButton,
+  Stack,
+} from '@mui/material';
```

Comment:
Unfortunately, since the site builds as a CommonJS module, it's best to import these separately, e.g.:
```
import Box from '@mui/material/Box';
import Button from '@mui/material/Button';
...etc...
```

Basically - importing directly from `@mui/material` indirectly pulls in the entire `@mui/material` package/everything in it since webpack with CommonJS modules can't treeshake unused code.

Further reading:
- https://webpack.js.org/guides/tree-shaking/
- https://mui.com/material-ui/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking
- We have a task to migrate to ESM so that treeshaking works as expected: https://hms-dbmi.atlassian.net/browse/CAT-124

Reply:
Ahh gotcha! That's good to know.